scripts/pegasus_rebalance_cluster.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+#
+# Pegasus cluster rebalance 
+#
+
+PID=$$
+
+if [ $# -le 2 ]; then
+  echo "USAGE: $0 <cluster-name> <cluster-meta-list> <only-move-primary>"
```

Comment:
```suggestion
  echo "USAGE: $0 <cluster-name> <cluster-meta-list> <only-move-primary>"
```

The second argument can be optional, default to false.